web/src/lib/components/shared-components/leaflet/asset-marker-cluster.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts" context="module">
+	import { createContext } from '$lib/utils/context';
+	import L from 'leaflet';
```

Comment:
Can we use `import {MarkerClusterGroup} from 'leaflet'` instead of importing the whole library?